ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml

```
@@ -1717,6 +1717,83 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )ocp-next-ui-e2e-tests,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

this change has to be made manually:
```suggestion
    always_run: false
```